### DISCUSSION TITLE
Sending a Batch of Images

### DISCUSSION THREAD

**Discussion by bely66** [bentoml/BentoML#2135] [Q&A]
So the main idea I understand in BentoML is that you can send a request at once.
or use micro-batching and batch multiple requests send by multiple users.

However in my case I want to send a batch of images to the API which isn't handled through any guide in the docs and I couldn't find any way to do it.

So if anyone has an idea on how to send a batch of images at once through the API, let me know.

**Comment by gregd33** [comment]
We played around with this too. 

The only good solution we found is to use the FileInput and accept a zip file of images. So in the service you unzip and process the images in batch. The client calling the service is responsible for zipping the images.
